src/main/java/org/isf/opd/manager/OpdBrowserManager.java

```diff
@@ -22,9 +22,8 @@
 package org.isf.opd.manager;
 
 import java.util.ArrayList;
-import java.util.Date;
-import java.util.GregorianCalendar;
 import java.util.List;
+import java.time.*;
```

Comment:
There are 8 occurrences of package imports; the standard is to enumerate all imports.  Thansk

Reply:
Correct: the standard is [here](https://github.com/informatici/openhospital-core#code-style).